src/main/java/org/opensearch/security/dlic/rest/validation/RequestContentValidator.java

```diff
@@ -286,14 +286,6 @@ public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params par
                         .get(SECURITY_RESTAPI_PASSWORD_VALIDATION_ERROR_MESSAGE, "Password does not match minimum criteria")
                 );
                 break;
-            case WEAK_PASSWORD:
```

Comment:
Will be picked up in the default statement

Reply:
I bit wrong here `ValidationError` has `WEAK_PASSWORD` and `SIMILAR_PASSWORD` errors.
AFAIU you want to get the same error message for all cases, so the solution in this case something like this:
```java

case INVALID_PASSWORD:
case WEAK_PASSWORD:
case SIMILAR_PASSWORD:
    builder.field("status", "error");
                builder.field(
                    "reason",
                    validationContext.settings()
                        .get(SECURITY_RESTAPI_PASSWORD_VALIDATION_ERROR_MESSAGE, "Password does not match minimum criteria")
                );
```
Otherwise you will end up in the `default` case:
which is: 
```java
    default:
                builder.field("status", "error");
                builder.field("reason", validationError.message());
                break;
```

Reply:
Here you can find different type of passwords to check:
https://github.com/opensearch-project/security/blob/6479513169e92a3781e9dd8da4ff32e0631b1bca/src/test/java/org/opensearch/security/dlic/rest/validation/PasswordValidatorTest.java#L30